typed-racket-lib/typed-racket/typecheck/tc-toplevel.rkt

```diff
@@ -271,6 +276,10 @@
        'no-type]
       ;; these forms we have been instructed to ignore
       [stx:ignore^
+       (cond
+         [(syntax-property #'stx 'tc-struct)
```

Comment:
Is it worth having a comment about why we don't ignore some syntax that's labeled as such?